src/main/java/com/github/wovnio/wovnjava/Api.java

```diff
@@ -51,8 +51,15 @@ String translate(String lang, String html) throws ApiException, ApiNoPageDataExc
               Proxy proxy = new Proxy(Proxy.Type.HTTP, new InetSocketAddress(this.settings.outboundProxyHost, this.settings.outboundProxyPort));
               con = (HttpURLConnection) url.openConnection(proxy);
             }
-            con.setConnectTimeout(settings.connectTimeout);
-            con.setReadTimeout(settings.readTimeout);
+            int connectTimeout = settings.connectTimeout;
+            int readTimeout = settings.readTimeout;
+            if (this.headers.isSearchEngineBot()) {
+                connectTimeout = this.settings.apiTimeoutSearchEngineBots;
+                readTimeout = this.settings.apiTimeoutSearchEngineBots;
+            }
+
+            con.setConnectTimeout(connectTimeout);
+            con.setReadTimeout(readTimeout);
```

Comment:
I noticed that wovnjava has separate settings for these two. I'm tempted to deprecate them in favour of a simpler `apiTimeout` setting. I don't think users particularly need to know about the difference between the connect/read timeout.